src/main/scala/group/scala/karazin/circe/literal/extras/macros.scala

```diff
@@ -56,7 +56,7 @@ object macros:
     private val ZoneOffsetUnit = java.time.ZoneOffset.MIN
 
     private val CurrencyUnit = java.util.Currency.getInstance("USD")
-    
+
     private val EitherKeysUnit = ("Left", "8lGfXcH5MnclW6gCbKrbP4ANYOQiNI9GK5futmz1")
```

Comment:
What is this second value for?

Reply:
Let's assume we have the following code:
```
extension (inline sc: StringContext)           
     inline def encode(inline args: Any*): Json =              
        ${ macros.encode[Either[Int, String]]('sc, 'args) }                       
   
val value: Either[Int, Boolean] = ???
                          
encode""" $value"""
```
In this case, our macro replaces $value with EitherUnitJson (e.g. { "Left": 0 } ).  In this case, it may happen that the unit is valid relative to the schema and there will be no compilation error. But in runtime, a value may appear that will not be valid (e. g. { "Right": true }).
Thus, it is necessary to validate two cases (left unit and right unit). For this, I created an invalid unit:
```
 {
  "Left": .... ,
  "8lGfXcH5MnclW6gCbKrbP4ANYOQiNI9GK5futmz1": ....
 }
```
And in the case when the macro validates the dynamically substituted value, we validate both units (left and right).

Reply:
I still didn't get why you're using 8lGfXcH5MnclW6gCbKrbP4ANYOQiNI9GK5futmz1 though. :)

Reply:
True, missed this point.